tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -203,12 +203,24 @@ copp/test_copp.py::TestCOPP::test_add_new_trap:
     conditions:
       - "is_multi_asic==True"
 
+  xfail:
+    reason: "Can't unisntall trap on broadcom 7050CX3 SKUs successfully"
```

Comment:
Is there an issue to track this failure, if yes, it's better to add it into the conditions, thanks

Reply:
The issue tracking this is currently private in the aristanetworks sonic-qual github project, should we open a public issue tracking the problem with install/uninstall trap on 7050CX3? @davidm-arista